threedi_modelchecker/tests/factories.py

```diff
@@ -194,22 +194,22 @@ class Meta:
     type = constants.Later2dType.SURFACE
 
 
-class ImperviousSurfaceFactory(factory.alchemy.SQLAlchemyModelFactory):
-    class Meta:
-        model = models.ImperviousSurface
-        sqlalchemy_session = None
+# class ImperviousSurfaceFactory(factory.alchemy.SQLAlchemyModelFactory):
+#     class Meta:
+#         model = models.ImperviousSurface
+#         sqlalchemy_session = None
+#
+#     surface_class = "pand"
+#     surface_inclination = "vlak"
+#     area = 0.0
```

Comment:
Do we want to keep these? Otherwise just remove them I think :)

Reply:
Oops, forgot to remove those. Fixed in a1c1ea71e2520351763d57d9e435b87ce08d791d